langserver/lsp.go

```diff
@@ -0,0 +1,95 @@
+package langserver
+
+type InitializeParams struct {
+	ProcessID             int                `json:"processId,omitempty"`
+	RootPath              string             `json:"rootPath,omitempty"`
+	InitializationOptions InitializeOptions  `json:"initializationOptions,omitempty"`
+	Capabilities          ClientCapabilities `json:"capabilities",omitempty`
+	Trace                 string             `json:"trace,omitempty"`
+}
+
+type InitializeOptions struct {
+}
+
+type ClientCapabilities struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type ClientCapabilities should have comment or be unexported